sample_templates/emr.yml.j2

```diff
@@ -0,0 +1,73 @@
+Auth:
+  region: "{{ region }}"
+Metrics:
+{% for emr in resources %}
+{% for metric_name in (
+    'AppsCompleted',
+    'AppsFailed',
+    'AppsKilled',
+    'AppsPending',
+    'AppsRunning',
+    'AppsSubmitted',
+    'CapacityRemainingGB',
+    'ContainerAllocated',
+    'ContainerPending',
+    'ContainerReserved',
+    'CoreNodesPending',
+    'CoreNodesRunning',
+    'CorruptBlocks',
+    'HDFSBytesRead',
+    'HDFSBytesWritten',
+    'IsIdle',
+    'MRActiveNodes',
+    'MRDecommissionedNodes',
+    'MRLostNodes',
+    'MRRebootedNodes',
+    'MRTotalNodes',
+    'MRUnhealthyNodes',
+    'MissingBlocks',
+    'PendingDeletionBlocks',
+    'S3BytesRead',
+    'S3BytesWritten',
+    'TotalLoad',
+    'UnderReplicatedBlocks',) %}
+- Namespace: "AWS/ElasticMapReduce"
+  MetricName: "{{ metric_name }}"
+  Statistics:
+  - "Average"
+  Unit: "Count"
```

Comment:
It isn't necessary to name then units, unless there are statistics that, for some reason, you don't want returned.